Diagnostics/AVTester/Test-ExchAVExclusions.ps1

```diff
@@ -104,205 +206,432 @@ if (-not($exchangeShell.ShellLoaded)) {
     exit
 }
 
-# Create the Array List
-$BaseFolders = Get-ExchAVExclusionsPaths -ExchangePath $ExchangePath -MsiProductMinor ([byte]$serverExchangeInstallDirectory.MsiProductMinor)
+Write-SimpleLogFile -String ("###########################################################################################") -name $LogFile
+Write-SimpleLogFile -String ("Starting AV Exclusions analysis at $((Get-Date).ToString())") -name $LogFile
+Write-SimpleLogFile -String ("###########################################################################################") -name $LogFile
 
-if ( $BaseFolders.count -eq 0 ) {
-    Write-Warning "We do not detect folders to analyze"
-    exit
-}
-
-# Create list object to hold all Folders we are going to test
-$FolderList = New-Object Collections.Generic.List[string]
-
-# Make sure each folders in our list resolve
-foreach ($path in $BaseFolders) {
+if ( -not $DisableProcessesAnalysis) {
     try {
-        # Resolve path only returns a bool so we have to manually throw to catch
-        if (!(Resolve-Path -Path $path -ErrorAction SilentlyContinue)) {
-            throw "Failed to resolve"
+        $response = $null
+        $response = Invoke-WebRequest http://crl.microsoft.com/pki/crl/products/CodeSigPCA.crl
+        if ($null -ne $response) {
+            if ( $response.StatusCode -eq 200 ) {
+                $Offline = $false
+            } else {
+                $Offline = $true
+            }
+        } else {
+            $Offline = $true
         }
-        # If -recurse then we need to find all SubFolders and Add them to the list to be tested
-        if ($Recurse) {
-
-            # Add the root folder
-            $FolderList.Add($path.ToLower())
+    } catch {
+        $Offline = $true
+    }
```

Comment:
We can make this simpler:
try {
        $response = $null
        $Offline = $true 
        $response = Invoke-WebRequest http://crl.microsoft.com/pki/crl/products/CodeSigPCA.crl
      $Offline = $null -eq $response -or $response.StatusCode -ne 200
      } catch {
      Write-verbose "Failed web request with inner error: $_"
       }